charts/vsphere-cpi/taints.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+export KUBECONFIG=$1
+for node in $(kubectl get nodes | awk '{print $1}' | tail -n +2)
+do
+	kubectl taint node $node node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule
```

Comment:
Does the vsphere operator handle removing this after it's initialized?

Reply:
Yes the cloud controller manager from CPI removes this taint once it's running

Reply:
If a new cluster is being created, if you set cloud-provider is set to external, that automatically taints all nodes with this taint. And once a cloud controller manager is installed, it is responsible for removing this taint.
But k8s does not set this taint on existing nodes even if you edit the cluster to set cloud-provider to external. Existing nodes need to be manually tainted, so this script only applies to the migration use case